### PR TITLE
Update XML to not embed quote in PATH on windows

### LINKED ISSUE #4335: Error in setting path on windows
I've installed the "Latest remote client for Windows"  tried it and uninstalled.

After uninstalling it the user path on my machine is left to `%PATH%;C:\Program Files (x86)\RedHat\Podman"` (in the registry under `HKEY_CURRENT_USER\Environment`) which causes problems both because I no longer have the paths set on my user path and because the `"` left in the path causes problems with other scripts

I'm not sure if the error happened during install or uninstall, if it is podman that causes it or the installer used on windows (it is the only new application I installed before having this problem), but it may be worth looking into it.

Thanks

Edited to add the reference to the registry

### PR DESCRIPTION
**Pull request by jwhonce**
* Remove the unsupported commend in lua script

Fixes #4335

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@giuseppe @vrothberg @mheon @TomSweeneyRedHat @baude PTAL

File: docs/links-to-html.lua [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# links-to-html.lua
```
Comment (TomSweeneyRedHat) [inline docs/links-to-html.lua]: Just triple checking, did you mean to remove this @jwhonce ?

**Pull request merged**